cash_input_common.c

```diff
@@ -17,13 +17,11 @@
  * limitations under the License.
  */
 
-#include <sys/poll.h>
-#include <sys/epoll.h>
 #include <errno.h>
 #include <stdbool.h>
 #include <pthread.h>
 #include <unistd.h>
-
+#include <pwd.h>
```

Comment:
Missing a `LOG_TAG`

Reply:
That's on purpose

Reply:
Errors from cash_input_comon.c don't show up when you grep logcat for `cash`. Setting a LOG_TAG here fixes that.

Reply:
The cash_input_common contains common code that is shared between all "modules" in CASH.
If there is any error coming from a cash_input_common function, unless this is failing on _all CASH modules_, then it's relative to a specific one.

For example, if I make a wrong call to a function in cash_input_common from cash_input_tof, that's a cash_input_tof error, not a common one, hence I want to see that the error comes from the ToF module.
Being it common, it could be happening on the RGBCIR module, as well, so if I see "CASH_INPUT_COMMON: Cannot allocate", I don't know if it comes from the ToF or TGBCIR module.

That's why I don't put a LOG_TAG here.

Reply:
Makes sense, thx

Reply:
I disagree. No `LOG_TAG` defined means that any logging printed from this compilation unit has an empty tag. As @ix5 correctly mentions, using a simple grep for `-i cash` now has missing lines while debugging.
If you want to see which module the error comes from, you should print an error in the respective module when a common function returns an error code. Having the logging in common itself aids in pinpointing the exact reason for that case (without deciphering the error code to print something useful in the tof/rgbcir module).

tl;dr: Either remove the common ALOG* calls altogether, or associate them with `cash`.